src/header.rs

```diff
@@ -64,6 +65,11 @@ pub struct Header {
     #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(rename = "x5t#S256")]
     pub x5t_s256: Option<String>,
+
+    /// Any additional headers
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(flatten)]
+    pub extra: Option<HashMap<String, String>>,
```

Comment:
Can the name be more self-explanatory? If we are to keep it short it should probably be `extras`.
I'm not sure what to call it. `extra` doesn't feel right.